<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="Rules">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>881</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>16</y>
      <width>231</width>
      <height>114</height>
     </rect>
    </property>
    <property name="title">
     <string>Действия</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEdit">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDelete">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>136</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Условия</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="premises"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>320</y>
      <width>231</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string>Вывод</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListWidget" name="condition"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuKnowledge">
    <property name="title">
     <string>Знания</string>
    </property>
    <addaction name="actionDomains"/>
    <addaction name="actionVariables"/>
    <addaction name="actionFacts"/>
    <addaction name="separator"/>
    <addaction name="actionViewOntology"/>
   </widget>
   <widget class="QMenu" name="menuRecommendation">
    <property name="title">
     <string>Рекомендация</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
   </widget>
   <widget class="QMenu" name="menuSolution">
    <property name="title">
     <string>Решение</string>
    </property>
    <addaction name="actionSolution"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuKnowledge"/>
   <addaction name="menuRecommendation"/>
   <addaction name="menuSolution"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Сохранить как ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Создать ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionVariables">
   <property name="text">
    <string>Переменные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDomains">
   <property name="text">
    <string>Домены</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Начать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionViewOntology">
   <property name="text">
    <string>Просмотр отнологии</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSolution">
   <property name="text">
    <string>Показать решение</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Цель консультации</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionFacts">
   <property name="text">
    <string>Факты</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
